core/src/main/java/jenkins/model/ItemCategory/ItemCategory.java

```diff
@@ -22,13 +22,12 @@
  * THE SOFTWARE.
  */
 
-package jenkins.model.item_category;
+package jenkins.model.ItemCategory;
```

Comment:
:bug: this package name violates Java naming conventions. Who asked for such a rename and why was it being done in this PR?